CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ You can find its changes [documented below](#030---2025-02-27).
 
 This release has an [MSRV] of 1.85.
 
+### Changed
+
+#### Parley
+
+- Breaking change: `Collection::register_fonts` now takes a `Blob<u8>` instead of a `Vec<u8>` ([#306][] by [@valadaptive])
```

Comment:
You'll need to add the link stuff below for both `[#306]` and `[@valadaptive]`. You can use the `[]` after your name as well, as you did for the PR link.

Just keep the names below alphabetically sorted. :)

Reply:
Ah, sneaky! I've added those now as well.